tests/functional_tests/find_a_legal_advisor/test_postcode_validation.py

```diff
@@ -67,25 +67,3 @@ def test_postcode_with_leading_spaces(self, page: Page, navigate_to_search) -> N
         expect(
             page.get_by_role("heading", name="Contact a legal adviser")
         ).to_be_visible()
-
-    def test_postcode_io_outtage(
```

Comment:
Was this removed intentionally?

Reply:
Yes, the way that the test is built causing significant issue in the test pipeline as restarting the live server removes other tests ability to access it. Agreed with Ben.